### DISCUSSION TITLE
Serial TTL Converter is not detected...

### DISCUSSION THREAD

**Discussion by NilsRo** [wmbusmeters/wmbusmeters#932] [General]
Hi,

I bought the following [TTL converter ](https://de.aliexpress.com/item/1005004667031804.html?spm=a2g0o.productlist.main.1.101c23e992hDuc&algo_pvid=43834f50-9712-4936-a3d3-7d883d183801&algo_exp_id=43834f50-9712-4936-a3d3-7d883d183801-0&pdp_ext_f=%7B"sku_id"%3A"12000030037735561"%7D&pdp_npi=3%40dis%21EUR%2139.2%2136.07%21%21%21%21%21%402100bfe316800417392423350d0703%2112000030037735561%21sea%21DE%212047527359&curPageLogUid=vvEzlKO11jSa) which is detected by Debian under /dev/ttyUSB0. But I cannot get it running connected multiple or only one Zenner C5. It does not detect the adapter: 
**(main) ignoring /dev/ttyUSB0, it does not respond as any of the supported wmbus devices.
No wmbus device detected, waiting for a device to be plugged in.**

As I understood a simple TTL converter should be supported and its detected by Linux with: Bus 005 Device 004: ID 067b:23d3 Prolific Technology, Inc. USB-Serial Controller

Checked/Planned
- [X] Autodetectiondevice=auto:t1
- [X] device=MAIN=/dev/ttyUSB0:mbus:2400
- [X] Bus Voltage
- [X] Connected one device only instead of 5
- [x] Oscilloscope

Is there perhaps something special to the meters that I have to enable MBus? Still changed the MBus Address in one meter from 0 to 1 for testing...any other ideas?

Cheers,
Nils

**Comment by weetmuts** [comment]
Its work now, right? :-)
https://github.com/wmbusmeters/wmbusmeters/issues/934

**Reply by NilsRo** [reply]
Yes, perfectly. 

The autodetection of ttyUSB0 seems not working or I misinterpreted something but setting it directly with the devices is working fine now.

**Reply by NilsRo** [reply]
> Its work now, right? :-) #934

I uses wired MBus. what would you suggest as a good poll interval? I read that devices should not polled that often to save battery life but is this also the case for wired MBus? The power supply of the interface is coming from the Bus so it is different from wMBus...  is the standard with 60s often or not?
